### PR TITLE
Unsubscribe Groups New Workflow

### PR DESCRIPTION
**Pull request by Whatthefoxsays**
**Description of the change**:
**Reason for the change**:
**Link to original source**:

Closes #

### CODE REVIEW

File: source/User_Guide/Transactional_Templates/how_to_send_an_email_with_transactional_templates.md [hunk 1]
```diff
@@ -149,7 +149,19 @@ You can also copy your template ID from the app and paste it into your call.
 Additional Resources
 {% endanchor h2 %}
 
+<<<<<<< HEAD
```
Comment (ksigler7) [inline source/User_Guide/Transactional_Templates/how_to_send_an_email_with_transactional_templates.md]: this all looks like it needs cleaned up in the source file

File: source/User_Guide/Transactional_Templates/how_to_send_an_email_with_transactional_templates_BACKUP_34922.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+---
+layout: page
```
Comment (ksigler7) [inline source/User_Guide/Transactional_Templates/how_to_send_an_email_with_transactional_templates_BACKUP_34922.md]: this page is a .backup and needs deleted

File: source/User_Guide/Transactional_Templates/how_to_send_an_email_with_transactional_templates_LOCAL_34922.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
+---
```
Comment (ksigler7) [inline source/User_Guide/Transactional_Templates/how_to_send_an_email_with_transactional_templates_LOCAL_34922.md]: and this is a .local!

File: source/User_Guide/Transactional_Templates/how_to_send_an_email_with_transactional_templates_REMOTE_34922.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
+---
```
Comment (ksigler7) [inline source/User_Guide/Transactional_Templates/how_to_send_an_email_with_transactional_templates_REMOTE_34922.md]: remote....

File: source/User_Guide/Transactional_Templates/how_to_send_an_email_with_transactional_templates_BASE_34922.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
+---
```
Comment (ksigler7) [inline source/User_Guide/Transactional_Templates/how_to_send_an_email_with_transactional_templates_BASE_34922.md]: this is a .base

File: source/User_Guide/Marketing_Campaigns/create-and-manage-unsubscribe-groups.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+Adding Unsubscribe Groups to your emails makes it easy to honor your recipients' email preferences and protects your sender reputation by complying with anti-spam legislation. 
+ 
+{% anchor h2 %}
+Create an Unsubscribe Group
+{% endanchor %}
+
+*To create an Unsubscribe Group:*
+
+1. Select **Marketing Campaigns** and then click **Unsubscribe Groups**.
```
Comment (sendgrid-ryan) [inline source/User_Guide/Marketing_Campaigns/create-and-manage-unsubscribe-groups.md]: Believe this should be "Select **Marketing**" per the nav.

File: source/User_Guide/Marketing_Campaigns/create-and-manage-unsubscribe-groups.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+1. Click **Add Unsubscribe Group**.
```
Comment (sendgrid-ryan) [inline source/User_Guide/Marketing_Campaigns/create-and-manage-unsubscribe-groups.md]: Add Unsubscribe Groups

File: source/User_Guide/Marketing_Campaigns/create-and-manage-unsubscribe-groups.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+
+{% info %}
+Make sure your Group Names and Group Descriptions are customer-friendly. This is what your recipients will see. 
+{% endinfo %}
+
+1. Select the checkbox if you want the Unsubscribe Group to display on the unsubscribe preferences page. 
+1. click **Save Unsubscribe Group**. 
+
+{% info %}
+To view the unsubscribe preferences page, select the action menu next to an Unsubscribe Group and then click **Preview**. Toggle to the Unsubscribe Preferences tab to view all of the options listed. {% endinfo %}
```
Comment (sendgrid-ryan) [inline source/User_Guide/Marketing_Campaigns/create-and-manage-unsubscribe-groups.md]: This update will now move the preview to be side by side as they are editing the fields, so this is no longer required for them to preview them. That said, once a group is created, they will be viewed as cards on the main Unsubscribe Groups index page, and there's now a 3 dot menu on each card, with a preview option as well.

File: source/User_Guide/Marketing_Campaigns/create-and-manage-unsubscribe-groups.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+Exporting an Unsubscribe Group List
+{% endanchor %}
+
+*To export an Unsubscribe Group List:*
+
+1. Navigate to the Unsubscribe Group you want to export.
+1. Click the action menu.
+1. Select **Export**. A CSV file begins downloading.
+
+[QUESTION - Does the account holder get an email notification like they do with MC Contacts?]
```
Comment (sendgrid-ryan) [inline source/User_Guide/Marketing_Campaigns/create-and-manage-unsubscribe-groups.md]: No

**Comment by thinkingserious** [comment]
Hello @Whatthefoxsays,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**